src/Compilers/Core/Portable/CaseInsensitiveComparison.cs

```diff
@@ -117,6 +117,24 @@ public override int Compare(string? str1, string? str2)
                 return str1.Length - str2.Length;
             }
 
+#if NETSTANDARD2_0 || NETCOREAPP
```

Comment:
Why are we not adding these APIs to desktop? Is one of the unicode APIs that we are using not available there?

Reply:
> Why are we not adding these APIs to desktop?

Desktop uses the **netstandard2.0** target, except for net20 targets that don't support these types.

Reply:
In that case let's use `#if !NET20` to make it clearer what's happpening here.

Reply:
There are also some netstandard 1.x targets that were impacted, but I can switch to a targeted exclusion.

Reply:
➡️ Switched to

```
#if !NET20 && !NETSTANDARD1_3
```